bin/start_gke

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# This is to be used by the Makefile for a start gke target.
+
+set -eo pipefail
+
+MY_IP=`curl icanhazip.com`
```

Comment:
For bash style, we generally prefer `$()` to ``` `` ```. There appears to be only one use of ``` `` ``` in the top-level `airflow.sh` file (side note: looks like this can probably be deleted since it appears to reference AWS).

As to using an arbitrary site to look up your public ip... I'd really prefer if we had a better system than this, but anything else (e.g. IAP and GKE, bastion hosts) are probably significantly more effort. I'd almost prefer making the cluster accessible from anywhere since there's additional auth at the k8s layer.

Reply:
I've been using this site for years without issue. If we want to open this cluster to the public, it may widen the surface area of attack vectors as this is a public repo with the projectid and other things visible.

Reply:
Since this is a development environment tool it's probably fine to rely on such a third-party service, but I'd request using `$(curl https://icanhazip.com)` and  a comment/link to https://major.io/icanhazip-com-faq/#what-about-my-privacy, as well as the option to override this via environment variable (i.e. only run this when the variable isn't set).

Again though, I'm not likely to use this tooling so if others are fine with it as-is I'm not going to object.